test/parallel/test-next-tick-when-exiting.js

```diff
@@ -1,14 +1,13 @@
 'use strict';
 
-require('../common');
+const common = require('../common');
 const assert = require('assert');
 
 process.on('exit', () => {
-  assert.strictEqual(process._exiting, true, 'process._exiting was not set!');
+  // process._exiting was not set!
```

Comment:
Nit: I think the comment is not needed.

Reply:
Sure, Actually this is my first PR, and not sure why windows test failed

I am a mac user, is there anything I can do to fix it?

Reply:
The failure is unrelated, don't worry about that.